### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by hsinewu**
---

- [ ] `./mach build -d` does not report any errors
- [ ] `./mach test-tidy` does not report any errors
- [ ] These changes fix #__ (github issue number if applicable).

- [ ] There are tests for these changes OR
- [ ] These changes do not require tests because _____

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13072)

### CODE REVIEW

File: tests/wpt/README.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ use the following:
 
     ./mach create-wpt --reftest tests/wpt/path/to/new/reftest.html --reference tests/wpt/path/to/reference.html
 
-`reference.html` will be created if it doesn't not exist, and `reftest.html`
+`reference.html` will be created if it doesn't exist, and `reftest.html`
```
Comment (KiChjang) [inline tests/wpt/README.md]: I actually prefer "does not" over "doesn't", since the former is more accepted as the written form.

**Comment by hsinewu** [comment]
Okay... XD

**Comment by nox** [comment]
@hsinewu I don't think @KiChjang meant that we don't want that PR, but that he would prefer "does not" to "doesn't". The documentation you changed did have a typo that should be fixed: "doesn't not" is incorrect.

**Comment by KiChjang** [comment]
this has been merged btw, in a different PR

**Comment by nox** [comment]
Oh ok, disregard me then.

**Comment by hsinewu** [comment]
@nox Thanks, I accidentally force push 0 commit to the branch, and then github close the issue automatically.
I wasn't aware of what's going on by that time, otherwise I would have try to fix this PR, instead of making a new one :/

**Pull request closed without merging**